Description: Fix string substitution issue with bash 4.4.
 Bash 4.4 returns an error when interpreting the string substitution
 "${#DIRArr[@]##*/}". This patch removes the part "##*/" of the
 substitution which does nothing in Bash pre-4.4.
Author: Ian Kelling <ian@iankelling.org>
Origin: https://github.com/graysky2/profile-sync-daemon/pull/185/commits/654f5359abfe7aab2e1616dd74c8ba0f76ce4ab4
Bug: https://github.com/graysky2/profile-sync-daemon/issues/182
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=839049
Forwarded: https://github.com/graysky2/profile-sync-daemon/pull/185
Last-Update: 2016-09-29
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/common/profile-sync-daemon.in
+++ b/common/profile-sync-daemon.in
@@ -356,7 +356,7 @@
 			# check that the LAST DIRECTORY in the full path is unique
 			unique_count=$(echo ${DIRArr[@]##*/} | sed 's/ /\n/g' | sort -u | wc -l)
 			# no problems so do nothing
-			[[ ${#DIRArr[@]##*/} -eq $unique_count ]] && continue
+			[[ ${#DIRArr[@]} -eq $unique_count ]] && continue
 
 			echo -e " ${RED}Error: ${NRM}${BLD}dup profile for ${GRN}$browser${NRM}${BLD} detected. See psd manpage, correct, and try again."${NRM}
 			# clip of the 'heftig-' to give correct path
